### PR TITLE
Fix shard_timeout environment variable definition and use some sane defaults

### PR DESCRIPTION
**Pull request by aidan-**
I had some interesting timeout/bad gateway issues using your container which turned out to be the default timeout of 60ms used for `request_timeout`.  I had a look in the `kibana.yml` that ships with kibana4-rc1 and they use much higher values as the default.

I also noticed the `shard_timeout` value is being populated with the wrong environment variable.

This PR fixes those two issues and has been tested.

### CODE REVIEW

**Comment by marcbachmann** [comment]
Thanks. That looks good. IMO a request timeout of 300 seconds is really high. But let's stick with the kibana defaults.

File: start.sh [hunk 1]
```diff
@@ -3,8 +3,8 @@
 export ELASTICSEARCH=${ELASTICSEARCH:-http://172.17.42.1:9200}
 export KIBANA_INDEX=${KIBANA_INDEX:-.kibana}
 export DEFAULT_APP_ID=${DEFAULT_APP_ID:-discover}
-export REQUEST_TIMEOUT=${REQUEST_TIMEOUT:-60}
```
Comment (marcbachmann) [inline start.sh:6]: Funny that this worked for me. :smile:

**Comment by marcbachmann** [comment]
After this merge, I have to update the RC1 tag. Can't do that on my phone.

File: start.sh (hunk 1 above)
Comment (aidan-) [inline start.sh:6]: It sometimes worked for me too ;) My requests were taking 58-62ms, which made this interesting to debug heh.

**Pull request merged**